src/backend/src/prisma/migrations/20221003023658_add_slack_id_param_to_user_settings/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "User_Settings" ADD COLUMN     "slackId" TEXT;
```

Comment:
as i said in my previous review, you need to delete this then run migrate again. right now this migration is not in line with what is in the prisma schema